examples/distributed-preconditioned-solver/doc/intro.dox

```diff
@@ -0,0 +1,8 @@
+<a name="Intro"></a>
+<h1>Introduction</h1>
+This distributed solver example should help you understand the basics of using Ginkgo in a distributed setting.
+The example will solve a simple 1D Laplace equation where the system can be distributed row-wise to multiple processes.
+To run the solver with multiple processes, use `mpirun -n NUM_PROCS ./distributed-solver [executor] [num_grid_points] [num_iterations]`.
+
+If you are using GPU devices, please make sure that you run this example with at most as many processes as you have GPU
+devices available.
```

Comment:
Maybe this should be rephrased. Since the example will work with more mpi ranks then GPUs.
```suggestion
If you are using GPU devices, please make sure that you run this example with at most as many processes as you have GPU
devices available, since oversubscription might lead to performance degradation
```